### DISCUSSION TITLE
Multiplatform support

### DISCUSSION THREAD

**Discussion by chrpai** [wixtoolset/issues#8028] [Questions]
I understand why WiX doesn't support multiplatforms but I also know Bob was looking for some kind of fast build / test feature.  I started thinking of the two in conjunction with each other.

I've recently started using a MacBook Pro as my daily driver as most of the developers in my company do the same.  We have repos with apps written in javascript, golang, python and .net  where the development occurs primarily on MacOS and deployment occurs on Windows.  To accomplish this I usually develop the WiX installer on a windows machine, submit the PR and set up a GitHub action that builds the app on Linux and then builds the MSI on Windows.

I've started thinking that if I ported my VS project templates to dotnet project templates and Migrated IsWiX over to MAUI that I'd have everything I need to setup a WiX project on MacOS.  The only thing missing is the ability to build.  But in thinking about it, I don't really need to build the MSI on Mac but rather just predict with a high level of confidence would the code build.   Is the XML well formed and validated against schemas.  Would the compiler throw any errors?  Would any files be expected to be failed to be found?  The full verification would be done during the build. 

Anyways just a thought and not a full formed WIP yet.  Any thoughts?  Would anyone see value in this?

If there's no appetite to do this in WiX, would anyone have an interest in a command line build tool that syncs your repos to a container in the cloud, builds it and syncs the output back as if it was built locally?

**Comment by barnson** [comment]
Rejected: https://github.com/wixtoolset/issues/issues/7823
